tutorials/Qa_skullstrip.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial for QA of Skull Strip"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is designed to illustrate the QA code used to generate the figure for the stripped brain and skull."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "this code will do 3 main things:\n",
+    "\n",
+    "1. Input original t1w file and the skull-striped brain file\n",
+    "2. Shows the skull-stripped brain (green) overlaid on the original t1w (magenta)\n",
+    "3. save the image into output directory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So that we can open the QA image to see if the skull strip result is good or not."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "#import packages\n",
+    "import warnings\n",
+    "\n",
+    "warnings.simplefilter(\"ignore\")\n",
+    "import sys\n",
+    "import nibabel as nib\n",
+    "import numpy as np\n",
+    "import os\n",
+    "from PIL import Image, ImageDraw,ImageFont\n",
+    "import matplotlib.pyplot as plt\n",
+    "from ndmg.stats.qa_skullstrip import gen_overlay_pngs"
+   ]
+  },
+  {
```

Comment:
Before "inputs" put a subheading called "gen_overlay_pngs"
underneath that heading put "The skullstrip qa images are created using the function gen_overlay_pngs, which (describe what function does)"
underneath that put a cell containing the code for gen_overlay_png

Then put a subheading called "plot_overlays_skullstrip"
underneath that put " skullstrip qa calls plot_overlays_skullstrip which which (describe what function does, in more than one sentence)"
the put a cell containing the code for plot_overlays_skullstrip

Reply:
https://github.com/neurodata/ndmg/pull/366/commits/65e3450758390ab8ed42ce8c64b32d8bacb71ae9